### ISSUE TITLE
Swedish JAS 39C/D loadout changes

### ISSUE THREAD

**Issue by og0172peli** [PygmalionOfCyprus/cmo-db-requests#3657]
### DB Selector

DB3K

### Affected DBID(s)

#3697 #3698

### Summary of Changes

JAS 39C/D is currently upgraded to be a part of the Swedish Air Force until at least the mid 2030's.

As such they will certainly use all available weapons that the Air Force has in it's current and future inventory including the new RB15 Mk4 Gungnir and the new AMRAAM 120-C8 (or D) missiles.

The BK 90 is not in use with the Swedish Air Force since 2012 so this should be removed from later JAS 39C/D/E/F versions and I don't think that the Rb 75 Maverick is in use longer as well, here I don't have a year.

Can you add a new JAS 39C/D version from 2023/2024 that includes loadouts with the RB15 Mk4 Gungnir and the AMRAAM 120-C8 (or D)?

### Sources

https://www.fmv.se/aktuellt--press/aktuella-handelser/uppgradering-av-gripen-cd2/

https://soff.se/medlemsnyhet/saab-lanserar-nasta-generations-sjomalsrobot-rbs15-gungnir-saabgroup/

https://www.dsca.mil/press-media/major-arms-sales/sweden-aim-120c-8-advanced-medium-range-air-air-missile-amraam

**Comment by PygmalionOfCyprus** [comment]
Please preface the DBIDs with an annex so they are properly parsed by the triage bot.

**Comment by Huib18** [comment]
Sweden bought the AIM-120C8 not the D variant
[https://www.dsca.mil/press-media/major-arms-sales/sweden-aim-120c-8-advanced-medium-range-air-air-missile-amraam](url)
[https://www.flightglobal.com/fixed-wing/sweden-approved-for-amraam-purchase-adding-to-surge-in-orders/154047.article](url)

**Comment by og0172peli** [comment]
> Please preface the DBIDs with an annex so they are properly parsed by the triage bot.

Ok, but I don't know how to du that now, I cant edit the post.

**Comment by og0172peli** [comment]
> Sweden bought the AIM-120C8 Is a C variant with a lot of things of the D A GPS satellite navigation system receiver and a two-way data transmission line complement the AIM-120C-8 (Formerly AIM-120D) guidance system. Compared to its previous version, the AIM-120C-7, it has a 50 per cent increased range, reaching about 160 km.
> 
> [https://www.dsca.mil/press-media/major-arms-sales/sweden-aim-120c-8-advanced-medium-range-air-air-missile-amraam](url) [https://www.flightglobal.com/fixed-wing/sweden-approved-for-amraam-purchase-adding-to-surge-in-orders/154047.article](url)

Yes I know that. The C-8 is not in the database, only C-7 and D.
